### PR TITLE
pom.xml: revert systemvm template version to 4.20.0.0

### PR DESCRIPTION
**Pull request by weizhouapache**
### Description

This PR reverts systemvm template version to 4.20.0.0

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI
- [ ] test (unit or integration test code)

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by weizhouapache** [comment]
@blueorangutan package

File: pom.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <project.systemvm.template.location>https://download.cloudstack.org/systemvm</project.systemvm.template.location>
-        <project.systemvm.template.version>4.20.1.0-SNAPSHOT</project.systemvm.template.version>
+        <project.systemvm.template.version>4.20.0.0</project.systemvm.template.version>
```
Comment (JoaoJandre) [inline pom.xml:53]: Could you explain @weizhouapache ? I followed https://cwiki.apache.org/confluence/display/CLOUDSTACK/Release+Procedure when setting this version. When should it change?

Comment (weizhouapache) [reply pom.xml:53]: > Could you explain @weizhouapache ? I followed https://cwiki.apache.org/confluence/display/CLOUDSTACK/Release+Procedure when setting this version. When should it change?

Until now, 4.20.1 and main/4.21 will still use the 4.20.0.0 system template on https://download.cloudstack.org/systemvm/4.20/

We can update the systemvm template version in pom.xml when we build new systemvm template later.

The cwiki page needs to be updated I think.

**Comment by weizhouapache** [comment]
@blueorangutan package

**Review by JoaoJandre — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (weizhouapache) [reply pom.xml:53]: > I see, I'll update it then. No problem

good, thanks !

**Comment by weizhouapache** [comment]
@blueorangutan test

**Review by DaanHoogland — APPROVED** [review]

**Pull request merged**